backend/cms/templates/settings/mfa/add_key.html

```diff
@@ -0,0 +1,137 @@
+{% extends "_raw.html" %}
+{% load i18n %}
+{% block raw_content %}
+{% load static %}
+{% load widget_tweaks %}
+<div id="no-session" class="flex flex-wrap flex-col justify-center px-3 py-10">
+    <div class="mx-auto w-full max-w-xs">
+        <div class="mb-3" style="padding: 0 50px;">
+            <a href="/">
+                <img src="{% static 'images/integreat-logo.png' %}" alt="{% trans 'Integreat Logo' %}" class="w-full" />
+            </a>
+        </div>
+        <div class="add-mfa bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4">
+            <form data-action="add2FaKey">
+                <div class="bg-red-lightest border-l-4 border-red text-red-dark px-4 py-3 mb-5 hidden add-mfa-error" role="alert">
+                    <p>{% trans 'Something went wrong during 2FA registration' %}</p>
+                    <p class="add-mfa-error-msg"></p>
+                </div>
+                <div class="mb-6">
+                    <p>{% trans 'Please enter a nickname for your new key. This nickname can be used to find the correct key in case you need to revoke authentication.' %}</p>
+                </div>
+                <div class="mb-4">
+                    <label class="block text-grey-darker text-sm font-bold mb-2" for="nickname">
+	                    {% trans 'Nickname' %}
+                    </label>
+	                {% trans 'Nickname' as nickname_placeholder%}
+                    {% render_field form.nickname id="nickname" placeholder=nickname_placeholder class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker leading-tight focus:outline-none focus:shadow-outline" %}
+                </div>
+                <div class="flex items-center justify-between">
+                    <button type="submit" class="bg-integreat hover:bg-grey-dark hover:text-white text-grey-darkest font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
+	                    {% trans 'Add 2FA key' %}
+                    </button>
+                </div>
+            </form>
+        </div>
+        <div class="mfa-not-supported bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 hidden">
+            <div class="mb-6">
+                <p>{% trans 'It seems like your browser does not support webauthn' %}</p>
+            </div>
+        </div>
+    </div>
+</div>
+{% endblock %}
+
+{% block javascript_nocompress %}
+<script>
+    window.django_csrf_token = '{{ csrf_token }}';
+</script>
+{% endblock %}
+{% block javascript %}
+<script>
+    // Based on https://github.com/duo-labs/py_webauthn/blob/master/flask_demo/static/js/webauthn.js
+    function b64enc(arrayBuffer) {
```

Comment:
Some JS code also seems to be duplicated, so if you want to you could outsource this code fragments to dedicated files and include them in the templates whenever necessary.